howso/utilities/installation_verification.py

```diff
@@ -422,26 +446,23 @@ def generate_dataframe(*, client: AbstractHowsoClient,
         }
     }
     feature_names = list(features.keys())
-    context_features = list(feature_names)[:-1]
-    action_features = [list(feature_names)[-1]]
 
-    trainee_obj = Trainee(
-        f"installation_verification generated dataframe ({get_nonce()})",
+    trainee = client.create_trainee(
+        name=f"installation_verification generated dataframe ({get_nonce()})",
         features=features,
         persistence="never"
     )
-    trainee = client.create_trainee(trainee_obj)
-    if not isinstance(trainee, Trainee):
+    if not isinstance(trainee, Dict):
         raise HowsoError('Unable to create trainee.')
-    client.set_feature_attributes(trainee.id, features)
-    client.acquire_trainee_resources(trainee.id, max_wait_time=0)
+    client.set_feature_attributes(trainee["id"], features)
+    client.acquire_trainee_resources(trainee["id"], max_wait_time=0)
     if timeout:
         # Generate 1 case at a time until `timeout` has passed.
         end_time = datetime.now() + timedelta(seconds=timeout)
         cases = {"action": []}
```

Comment:
Erroneous tildas at end of this line
```suggestion
        end_time = datetime.now() + timedelta(seconds=timeout)
```

Reply:
Resolved by prior commits.